docs/_manage/invite-team-member.md

```diff
@@ -81,3 +81,31 @@ To sign up as a collaborator, log in to Zapier (or sign up for Zapier if you don
 ![JoinIntegrationFromDashbaord](https://cdn.zappy.app/5a3452ea8e72049a2083aab5dac59069.png)
 
 ![JoinIntegration](https://cdn.zappy.app/b47f3a9b672533df505667cf3933a9ee.png)
+
+## Managing Partner Email Subscriptions
+
+### General Partner Marketing Updates
+
+Zapier sends out general partner marketing updates related to your integration and the Zapier platform. These emails include:
+* Monthly Partner Newsletter with integration data insights
+* Outreach about partnership and co-marketing opportunities 
+
+To control your subscription for these emails, visit the [email preferences page](https://developer.zapier.com/partner-settings/email) in the developer dashboard and opt in or out per integration as needed.
+
+### Platform and Partner Program Updates
+
+We'll keep you in the loop with Zapier Platform and Partner Program Updates. These emails include:
+* Alerts about user [Bugs and Feature Requests](https://platform.zapier.com/manage/analyze-integration-performance#bugs)
+* Critical Platform and Program Updates
```

Comment:
Does "platform and program updates" need to be capitalized? Maybe it's the name of the email you're referring to?